scripts/spack/packages/umpire/package.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
Were there any edits made to this one? I didn't see any `SERAC_EDIT_BEGIN`

Reply:
This was pulled in because the spack sha we were on didnt have the new umpire version.